sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -54,11 +55,10 @@ def test_row_as_dict(self):
     self.assertEqual(test_value, coder.decode(coder.encode(test_value)))
 
   def json_compliance_exception(self, value):
-    with self.assertRaises(ValueError) as exn:
+    with self.assertRaisesRegexp(ValueError, JSON_COMPLIANCE_ERROR):
```

Comment:
<!--new_thread; commit:4709161da347f4fa5eedbd7f3c09911b573fadac; resolved:0-->
This is fragile since this is not designed to be used as an regexp.  How about:

```python
'^' + re.escape(JSON_COMPLIANCE_ERROR)
```

Reply:
done